### PR TITLE
Update install for armhf install

### PR DESCRIPTION
**Pull request by rosbeef**
## Problem

- install/upgrade/... not working on armhf

## Solution

-  work on armhf
## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

### CODE REVIEW

**Review by MayeulC — CHANGES_REQUESTED** [review]
Thank you for the report. I am not sure this is the cleanest solution, however. And there seems to be unrelated changes too? Could you comment on why there are necessary?

**Comment by rosbeef** [comment]
hi @MayeulC i did changes and i tested installation and it works for me

File: scripts/install [hunk 1]
```diff
@@ -148,12 +148,28 @@ chown $app:$app "$final_path/config.yaml"
...
 
+if [ $YNH_ARCH == "armhf" ] || [ $YNH_ARCH == "armel" ]
+then
+# Install rustup is not already installed
+# We need this to be able to install cryptgraphy
+export PATH="$PATH:$final_path/.cargo/bin:$final_path/.local/bin:/usr/local/sbin"
+    if [ -e $final_path/.rustup ]; then
+    sudo -u "$app" env PATH=$PATH rustup update
+    else
+    sudo -u "$app" bash -c 'curl -sSf -L https://static.rust-lang.org/rustup.sh | sh -s -- -y --default-toolchain=stable --profile=minimal'
```
[the remark this answers is not in the document]
Comment (gautz) [reply scripts/install:159]: ping @tituspijean @ericgaspar do you think this makes sense ? Would packaging v2 solve this armhf install issue in a cleaner way ?

**Comment by nathanael-h** [comment]
!test-me

**Comment by nathanael-h** [comment]
!testme

**Pull request merged**